repo/packages/D/dcos-enterprise-cli/16/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-enterprise-cli",
+  "version": "1.4.0",
```

Comment:
This doesn't match with the endpoint URL below, which is still 1.3.0.

Reply:
I see. I should push a change which puts this at 1.3.0 then?

Reply:
The last `dcos-enterprise-cli` package in universe is already 1.3.0, we should bump it to `1.3.1` or `1.4.0`. Though as you don't want to backport this I suggest we wait for the licensing command integration, then your changes in the backup CLI will be included altogether for DC/OS `1.11`.

Reply:
@bamarni do you mind making whatever changes make sense here, given that we don't really need to backport it to 1.10?

Reply:
I'll open a new PR here when the licensing command will be ready, closing for now.